src/mixins/mixin.js

```diff
@@ -105,15 +105,21 @@ export default {
         let data = payload.data
 
         switch (op) {
+          // notebook
+          case 'NEW_NOTE':
+            window.open(`/notebook/${data.note.id}`, '_blank')
```

Comment:
This will open it in a new tab for now right?. This is fine for now let's make a left panel and show a list of all notebooks there, and users can also add folders and create notebooks inside them.

cc @liuxunorg

Reply:
@ieglonew01f I agree with that. Let's make the left panel